modules/central-services-private-config.adoc

```diff
@@ -12,15 +12,18 @@ There are no default values for these parameters.
 == Image pull secrets
 The credentials required for pulling images from your registry.
 
-[NOTE]
-====
-* If you are using the `stackrox.io` registry, you must specify the `imagePullSecrets.username` and `imagePullSecrets.password` parameters.
-* If you are using a custom registry, you must also specify the `image.registry` parameter.
-* If you do not use a username and password to log in to your custom registry, you must specify one of the following parameters:
-** `imagePullSecrets.allowNone`
-** `imagePullSecrets.useExisting`
-** `imagePullSecrets.useFromDefaultServiceAccount`
-====
+If you are using a custom registry, you must specify these parameters:
+
+* `imagePullSecrets.username`
+* `imagePullSecrets.password`
+* `image.registry` 
+
+If you do not use a username and password to log in to your custom registry, you must specify one of the following parameters:
```

Comment:
Suggesting to rephrase it a bit, optionally. Primarily thinking about enterprises where the person installing ACS is not controlling the image registry therefore "your custom registry" does not apply.

```suggestion
If you do not use a username and password to log in to the custom image registry, you must specify one of the following parameters:
```